sw/device/examples/sram_program/BUILD

```diff
@@ -31,6 +32,42 @@ opentitan_ram_binary(
         "//hw/top_earlgrey/sw/autogen:top_earlgrey",
         "//sw/device/lib/base:macros",
         "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing:pinmux_testutils",
         "//sw/device/lib/testing/test_framework:check",
     ],
 )
+
+opentitan_gdb_fpga_cw310_test(
+    name = "sram_program_fpga_cw310_test",
+    exit_success_pattern = "sram_program\\.c:47\\] PC: 0x1000011c, SRAM: \\[0x10000000, 0x10020000\\)",
```

Comment:
This success pattern should be updated now that sram_program is built for a higher address.

Reply:
on my machine:
```
sram_program.c:40] PC: 0x1000202c, SRAM: [0x10000000, 0x10020000)
```